src/components/DesignTheme/View/UOView.jsx

```diff
@@ -172,7 +172,7 @@ const UOView = ({ content }) => {
                 </div>
               </article>
             ) : null}
-            {content.assessore_riferimento.length > 0 ? (
+            {content.assessore_riferimento ? (
```

Comment:
Questo genere di controlli è rischioso. Se ho capito bene `assessore_riferimento` è un array. Gli array in javascript sono true-ish anche se vuoti. In questo caso se l'array è vuoto ti stampa comunque il titolo. L'altro lato della medaglia è che il controllo `.length` si spacca se lo fai su undefined o null, ovviamente. Forse farei `content.assessore_riferimento?.length > 0` per tagliare la testa al toro.

Reply:
Sto pensando al perché l'ho modificato così: passando da .length al solo ?.
Mi pare di ricordare che mi tornasse null quel campo vuoto, perché è un multiselect, ma limitato ad 1... bho. In effetti torna la lista. Lo schema in backend è coerente con quello che torna. Concordo sul ?.length